dkms.in

```diff
@@ -1567,41 +1535,37 @@ do_uninstall()
             fi
         fi
 
+        echo $""
         for ((count=0; count < ${#built_module_name[@]}; count++)); do
             real_dest_module_location="$(find_actual_dest_module_location $module $count $1 $2)"
-            echo $""
-            echo $"${dest_module_name[$count]}$module_suffix:"
-            echo $" - Uninstallation"
+
             if [[ ${real_dest_module_location} ]]; then
-                echo $"   - Deleting from: $install_tree/$1${real_dest_module_location}/"
+                echo $"Deleting $install_tree/$1${real_dest_module_location}/${dest_module_name[$count]}$module_suffix"
                 rm -f "$install_tree/$1${real_dest_module_location}/${dest_module_name[$count]}$module_uncompressed_suffix"*
                 dir_to_remove="${real_dest_module_location#/}"
                 while [[ ${dir_to_remove} != ${dir_to_remove#/} ]]; do
                     dir_to_remove="${dir_to_remove#/}"
                 done
                 (if cd "$install_tree/$1"; then rpm -qf "${dir_to_remove}" >/dev/null 2>&1 || rmdir -p --ignore-fail-on-non-empty "${dir_to_remove}"; fi || true)
             else
-                echo $"   - Module was not found within $install_tree/$1/"
+                echo $"Module was not found within $install_tree/$1/"
             fi
-            echo $" - Original module"
             local origmod=$(compressed_or_uncompressed "$dkms_tree/$module/original_module/$1/$2" "${dest_module_name[$count]}")
             if [[ -n $origmod ]]; then
                 case "$running_distribution" in
                     debian* | ubuntu* )
```

Comment:
This seems inconsistent with the change earlier. Shouldn't we also restore the modules if we backed them up on debian/ubuntu?

Reply:
Good catch! Let me remove the empty `case`.